sirius-pipeline/SAI/sai_api_gen.py

```diff
@@ -4,85 +4,82 @@
     import os
```

Comment:
Wishlist item: a test for the sai_api_gen.py as a sanity check to catch a breaking change. We could snapshot a golden set of input and output files or something similar.